chainercv/links/model/faster_rcnn/utils/anchor_target_creator.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+
+import chainer
+from chainer import cuda
+
+from chainercv.links.model.faster_rcnn.utils.bbox2loc import bbox2loc
+from chainercv.utils.bbox.bbox_iou import bbox_iou
+
+
+class AnchorTargetCreator(object):
+
+    """Assign the ground truth bounding boxes to anchors.
+
+    Assigns the ground-truth bounding boxes to anchors for training Region
+    Proposal Networks introduced in Faster R-CNN [#]_.
+
+    Offsets and scales to match anchors to the ground truth are
+    calculated using the encoding scheme of
+    :obj:`chainercv.links.model.faster_rcnn.bbox2loc`.
+
+    .. [#] Shaoqing Ren, Kaiming He, Ross Girshick, Jian Sun. \
+    Faster R-CNN: Towards Real-Time Object Detection with \
+    Region Proposal Networks. NIPS 2015.
+
+    Args:
+        n_sample (int): The number of regions to produce.
+        positive_iou_thresh (float): Anchors with IoU above this
+            threshold will be assigned as positive.
+        negative_iou_thresh (float): Anchors with IoU below this
+            threshold will be assigned as negative.
+        fg_fraction (float): Fraction of positive regions in the
+            sampled regions.
```

Comment:
In this function, you are using both `positive/negative` and `foreground/background`. Are they different from each other?